public/svgs/hamburger.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="28" height="26" viewBox="0 0 28 26" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Add these as Icons, see `icons/index.ts`

Reply:
Ok, just a question, why we don't use the actual icons from material-icons, for hamburger and close icon? @AlexDistill